packages/ui-core/src/components/slider/slider.styles.ts

```diff
@@ -0,0 +1,160 @@
+import styled from 'styled-components';
+
+export const TickContainer = styled.div<{
+  left: number;
+  controlSize: number;
+  type: 'horizontal' | 'vertical';
+}>`
+  position: absolute;
+  width: 30px;
+
+  ${props =>
+    props.type === 'horizontal' &&
```

Comment:
`
import { css } from 'styled-components'
 props.type === 'horizontal' && css``
`

Reply:
Instead of raw string